src/pages/hacker-manual.jsx

```diff
@@ -110,7 +135,7 @@ const About = () => (
           please be wary of the risks and potential scams.
         </p>
         <p>
-          We recommend the districts Treptow, Kreuzberg, Neukölln, or
+          We recommend the districts Treptow, Kreuzberg, Neuk&ouml;lln, or
```

Comment:
Just curious are you manually and explicitly changing this (if so why?) or does your IDE do it automatically?

Reply:
I do it whenever I see it. I think it shows I'm very old. Back in the day you couldn't use umlauts in the source.